user/test.py

```diff
@@ -0,0 +1,46 @@
+import os
+from unittest import TestCase
+
+from user.models import User
+from blog.models import Article
+from trips import views
+from around import views
+from core import app
+
+
+class UserTest(TestCase):
+    def setUp(self):
+        self.client = app.test_client()
+        self.user = User(email='emailtest@test.com').set_password('testtest')
+        self.user.save()
+
+    def tearDown(self):
+        User.objects.delete()
+        Article.objects.delete()
+
+    def test_unpublished_articles_does_not_appears_if_the_author_not_logged(
```

Comment:
I believe you could move any test about article in the _blog_ module.
Let's imagine the project would someday see the _blog_ module removed,  these tests should be removed as well. (not planned hu, just exposing modules responsibilities).